src/components/templates/PosterPage/PosterPage.tsx

```diff
@@ -183,6 +183,7 @@ export const PosterPage: React.FC<PosterPageProps> = ({ venue }) => {
             title={title}
             allow={IFRAME_ALLOW}
             allowFullScreen
+            frameBorder="0"
```

Comment:
I’m not 100% off the top of my head but I’m pretty sure this is a legacy prop and this should be done with CSS these days.

Reply:
done! commit was rewritten and force pushed

Reply:
Thanks :)